### ISSUE TITLE
/machine_learning/deepdive2/introduction_to_tensorflow/labs/write_low_level_code.ipynb

### ISSUE THREAD

**Issue by ffyring** [GoogleCloudPlatform/training-data-analyst#1179]
It is of no consequence, but bonus section has an error: 

`def compute_gradients(X, Y, W):
    with tf.GradientTape() as tape:
        loss = loss_mse(Xf, Y, W)      # <== Should be X
    return tape.gradient(loss, W)
`

//Fredrik
